src/pages/tokens/[tokenid].js

```diff
@@ -315,11 +315,16 @@ export default function Token(props) {
                 }}
               >
                 <Icon icon={CalendarIcon} />
-                {token.created_at !== null
-                  ? `Minted on ${moment(tree.time_created).format(
-                      'MMMM Do, YYYY',
-                    )}`
-                  : 'Unknown Mint Date'}
+                {token.created_at !== null ? (
+                  <>
+                    Minted on
+                    <time dateTime={moment(tree.time_created)}>
```

Comment:
No need to pass a moment Date object to the `dateTime` prop the `tree.time_created` is good

```suggestion
                    <time dateTime={tree.time_created}>
```